lib/posix/Kconfig.semaphore

```diff
@@ -8,3 +8,10 @@ config SEM_VALUE_MAX
 	range 1 32767
 	help
 	  Maximum semaphore count in POSIX compliant Application.
+
+config SEM_NAMELEN_MAX
+	int "Maximum name length"
+	default 16
```

Comment:
BTW, regarding the pointer vs array tradeoffs. If we set the default to 6, then there would be no penalty when comparing dynamic and static allocation sizes. Anyway, this was just a note, no need to change this.

Reply:
It might be good to note in the `help` section, is that the value 255 corresponds to `NAME_MAX`.

Reply:
> It might be good to note in the `help` section that the value 255 corresponds to `NAME_MAX`.

Fixed